lib/puppet/configurer.rb

```diff
@@ -230,10 +230,10 @@ def run(options = {})
             raise Puppet::Error, _("Could not select a functional puppet master from server_list: '%{server_list}'") % { server_list: Puppet.settings.value(:server_list, Puppet[:environment].to_sym, true) }
           else
             #TRANSLATORS 'server_list' is the name of a setting and should not be translated
-            Puppet.debug _("Selected puppet server from the `server_list` setting: %{server}:%{port}") % { server: server, port: port }
+            Puppet.debug _("Selected server from the `server_list` setting: %{server}:%{port}") % {server: server, port: port}
             report.master_used = "#{server}:#{port}"
           end
-          Puppet.override(server: server, serverport: port) do
+          Puppet.override(:server => server, :serverport => port) do
```

Comment:
6.0.x and up say "Selected puppet server..." and use `server: server`. I think we should PR 5.5.x to match so that we don't have this merge conflict.